pages/documentation/maturity-model.md

```diff
@@ -29,11 +29,10 @@ subnav:
       </ul>
       </div>
     </div>
-    <div class="tablet:grid-col-5">
-      <div class="padding-2 tablet:padding-left-4">
-        <img src="{{ site.baseurl }}/img/maturity-model.svg" role="img" alt="A diagram showing the concentric circles of the USWDS maturity model, progressing from Principles on the outside, through Guidance, and finally to Code in the innermost circle.">
-      </div>
-    </div>
+    <figure class="site-figure tablet:grid-col-5 padding-2 tablet:padding-left-4">
+      <img src="{{ site.baseurl }}/img/maturity-model.svg" role="img" alt="A diagram showing the concentric circles of the USWDS maturity model, progressing from Principles on the outside, through Guidance, and finally to Code in the innermost circle.">
+      <figcaption class="site-figcaption">In this image you can see 3 levels of "maturity" in relation to the U.S. Web Design System. As you can see, you don't have to adopt the code to adopt the guidance and benefits outlined by the Design System</figcaption>
```

Comment:
Should the `3` be  spelled out here? Relying on my college AP style class here, so let me know if that isn't what we do here :disguised_face:

Also, found a missing a period at the end. 

```suggestion
      <figcaption class="site-figcaption">In this image you can see three levels of "maturity" in relation to the U.S. Web Design System. As you can see, you don't have to adopt the code to adopt the guidance and benefits outlined by the Design System.</figcaption>
```